docker/hab2_tpsrl.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM fairembodied/habitat-challenge:habitat_rearrangement_2022_base_docker
+ADD agents/habitat_baselines_agent.py agent.py
+ADD configs/ /configs/
+RUN /bin/bash -c "mkdir -p data/models"
+ADD data/models/nav.pth data/models/nav.pth
```

Comment:
Do you want to put these files on our S3 bucket instead? It is easier to download from there and incase someone wants to share new files in the future, google drive links cant be shared with public from inside meta.

Reply:
I think these model checkpoints are 1-offs for the challenge, they are not compatible with `main` anymore that's why I thought it made more sense to keep as a google drive link. However, I can change if necessary.

Reply:
Let's put them on S3 with a name that includes a version and suggests that these models will not be long-lived.

Reply:
Ok sure, I will put these models on S3 soon.